woocommerce-paypal-payments.php

```diff
@@ -229,4 +229,11 @@ function is_woocommerce_activated(): bool {
 		return class_exists( 'woocommerce' );
 	}
 
+	// Set new merchant flag on plugin install.
+	add_action( 'woocommerce_paypal_payments_gateway_migrate', function() {
```

Comment:
We could also use the `$installed_version` parameter to detect first installation:

```php
add_action( 'woocommerce_paypal_payments_gateway_migrate', function( $version ) {
  if ( false !== $version ) {
    return;
  }

  update_option( 'woocommerce-ppcp-is-new-merchant', true );
});
```